span.go

```diff
@@ -60,21 +73,38 @@ func (e *traceExporter) convertSpan(s *trace.SpanData) *ddSpan {
 	if s.ParentSpanID != (trace.SpanID{}) {
 		span.ParentID = binary.BigEndian.Uint64(s.ParentSpanID[:])
 	}
+
+	httpCode := httpCode(s.Status.Code)
 	switch s.SpanKind {
 	case trace.SpanKindClient:
 		span.Type = "client"
+		if httpCode >= 400 && httpCode < 500 {
```

Comment:
I'm wondering if it's nicer and easier to reason about as:
```go
if code/100 == 4 {
```
🤔

Reply:
👍